src/uct/ib/mlx5/dc/dc_mlx5.c

```diff
@@ -79,7 +79,10 @@ ucs_config_field_t uct_dc_mlx5_iface_config_sub_table[] = {
      "           Multiple endpoints may share the same DCI.\n"
      "\n"
      "hw_dcs     A single DCI that operates as a HW DCS queue. The channels are assigned\n"
-     "           in a round-robin fashion.",
+     "           in a round-robin fashion."
+     "\n"
+     "dcs_hybrid Same as \"dcs_quota\" but when there are no available dcis in the pool,\n" 
+     "           using a dedicated HW dci which will be used the same way as in hw_dcs policy.",
```

Comment:
dci -> DCI

Reply:
+1

Reply:
`hw_dcs` -> `\"hw_dcs\"`

Reply:
Rephrase and simplify:
`, but when there are no DCIs available, a dedicated HW DCI is used in the same manner as..`